addon.gypi

```diff
@@ -109,7 +109,7 @@
           '-luuid.lib',
           '-lodbc32.lib',
           '-lDelayImp.lib',
-          '-l"<(node_root_dir)/$(ConfigurationName)/<(node_lib_file)"'
+          '-l<(node_root_dir)/<(CONFIGURATION_NAME)/<(node_lib_file)'
```

Comment:
won't this stop working for `MSBuild` ? Where is `(CONFIGURATION_NAME) defined ?